### PR TITLE
upload logs to gcs after failure

### PR DESCRIPTION
**Pull request by jenniexie**
Hi @gerickson @jaylogue @robszewczyk :
   This is the PR to address issue of "not uploading logs after test failure", please help to review.
I have tested it on one of my test branch, which I did return value 2 by intention to make test fail everytime:
https://github.com/openweave/openweave-core/tree/lwip_upload_fail_log
result link for test branch when test failedis:
https://travis-ci.org/openweave/openweave-core/builds/557980883

Thanks
Jennie

### CODE REVIEW

**Comment by jenniexie** [comment]
I am testing this PR by changing return value to 2, so test will fail and prove it can upload test logs after test fail
I will remove "return 2" after test proved work.

**Comment by jenniexie** [comment]
> I am testing this PR by changing return value to 2, so test will fail and prove it can upload test logs after test fail
> I will remove "return 2" after test proved work.

@jaylogue @robszewczyk, I have confidence for uploading logs after test fail, prove is: build #1409 https://travis-ci.org/openweave/openweave-core/builds/558090243
uploaded logs to gcs:
https://pantheon.corp.google.com/storage/browser/openweave/happy-test-log/?project=openweave-test-project&authuser=0&organizationId=433637338589

next step:
 I will remove "return 2" now and we can merge it to master.

**Comment by jenniexie** [comment]
I have removed "return 2" from ./travis/script.sh, and changed the gcs key to a new encrypted key. 
The PR is ready for merge now.
https://github.com/openweave/openweave-core/pull/305

File: .travis.yml [hunk 1]
```diff
@@ -33,6 +33,9 @@ script:
 after_success:
   - bash <(curl -s https://codecov.io/bash)
 
+after_failure:
+  - ".travis/after_failure.sh"
```
Comment (gerickson) [inline .travis.yml]: No quotes needed.

Comment (jenniexie) [reply .travis.yml]: Good eyes:), Thanks Grant!
I just pushed the fix for it

**Pull request merged**